### PR TITLE
Rust 1.70

### PR DESCRIPTION
**Pull request by scottlamb**

### CODE REVIEW

**Comment by lzuosym** [comment]
some `variable does not need to be mutable` warnings from running `cargo check`
https://github.com/sportsball-ai/av-rs/blob/main/srt/src/lib.rs#L328
https://github.com/sportsball-ai/av-rs/blob/main/srt/src/lib.rs#L335

**Review by lzuosym — CHANGES_REQUESTED** [review]

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ jobs:
     - name: Install Rust
       uses: actions-rs/toolchain@v1
       with:
-        toolchain: 1.63.0
+        toolchain: 1.70.0
```
[the remark this answers is not in the document]
Comment (scottlamb) [reply .github/workflows/rust.yml:30]: Good catch; will do.

**Comment by scottlamb** [comment]
> some variable does not need to be mutable warnings from running cargo check

Strange. I don't see those:

```
[slamb@slamb-txm ~/git/sportsball-ai/av-rs/srt]$ cargo +1.70 clippy --all-features
    Finished dev [unoptimized + debuginfo] target(s) in 0.11s
[slamb@slamb-txm ~/git/sportsball-ai/av-rs/srt]$ cargo +1.70 check --all-features
    Finished dev [unoptimized + debuginfo] target(s) in 0.20s
```

I don't see it in the CI output either.

**Comment by scottlamb** [comment]
> I am getting a warning some crates are on edition 2021 which defaults to resolver = "2", but virtual workspaces default to resolver = "1"``

What command produces that? I don't see it.

and yes I can do the resolver change.

**Comment by lzuosym** [comment]
> Strange. I don't see those:
> 
> ```
> [slamb@slamb-txm ~/git/sportsball-ai/av-rs/srt]$ cargo +1.70 clippy --all-features
>     Finished dev [unoptimized + debuginfo] target(s) in 0.11s
> [slamb@slamb-txm ~/git/sportsball-ai/av-rs/srt]$ cargo +1.70 check --all-features
>     Finished dev [unoptimized + debuginfo] target(s) in 0.20s
> ```
> 
> I don't see it in the CI output either.

I see the warnings in the nightly version rather than in 1.70. We shouldn't concern with it.

**Review by lzuosym — APPROVED** [review]

**Pull request merged**